docs/CodingStandards.md

```diff
@@ -0,0 +1,12 @@
+# Coding Standards
+
+Note that some projects might have addition guidelines for the languages they use.
+
+* C#
+* CSS
+* **Java**: [[Style guide](https://oss-generic.github.io/process/codingstandards/coding-standards-java.html)]
+  [Best practices]
```

Comment:
I think the wrapping `[...][Best practices]` is unnecessary. Similarly for the HTML one.